Tasks/VsTest/vstest.ts

```diff
@@ -187,6 +187,28 @@ function generateResponseFile(): Q.Promise<string> {
     return defer.promise;
 }
 
+function publishCodeChanges(): Q.Promise<string> {
+    var defer = Q.defer<string>();
+    var selectortool = tl.createToolRunner(tiaSelectorTool);
+    selectortool.arg("PublishCodeChanges");
+    selectortool.arg("/TfsTeamProjectCollection:" + tl.getVariable("System.TeamFoundationCollectionUri"));
+    selectortool.arg("/ProjectId:" + tl.getVariable("System.TeamProject"));
+    selectortool.arg("/newdropPath:" + tl.getVariable("Build.SourcesDirectory"));
+    selectortool.arg("/Definitionid:" + tl.getVariable("System.DefinitionId"));
+    selectortool.arg("/buildid:" + tl.getVariable("Build.BuildId"));
+    selectortool.arg("/token:" + tl.getVariable("System.AccessToken"));
+
+    selectortool.exec()
```

Comment:
The documentation of exec code says this:
"Exec a tool.
     * Output will be streamed to the live console."
Exec will take care of the output